consonance-arch/src/main/java/io/consonance/arch/worker/WorkerRunnable.java

```diff
@@ -218,6 +234,78 @@ public void run() {
                 // Do the actual work
                 processJobMessage(workflowResult, statusJSON, job);
 
+                // create the job exchange
```

Comment:
It was redundant, forgot to remove that somehow when I merged your changes. Waiting for checks now.